CODEOWNERS

```diff
@@ -17,23 +17,18 @@ azure-cron.yml      @garyverhaegen-da
 /release.sh         @garyverhaegen-da
 
 # Blackduck
-NOTICES     @garyverhaegen-da @cocreature @dasormeter
+NOTICES     @garyverhaegen-da @dasormeter
 
 # Language
-/compiler/          @cocreature
-/libs-haskell/      @cocreature
-/daml-assistant/    @sofiafaro-da @cocreature
-/daml-script/       @cocreature
-/triggers/          @cocreature
-/ghc-lib/           @cocreature
-/compatibility/     @garyverhaegen-da @cocreature
```

Comment:
@adriaanm-da should anyone from the language team take over those deleted lines?

(For context: this is not enforced in any way, it's just about getting notifications when those files change.)

Reply:
I pinged @sofiafaro-da and @remyhaemmerle-da already. I’ll let them decide if and which ones.

Reply:
I'll take these ones under "Language". Happy to do it in a separate PR though.

Maybe @ray-roestenburg-da or someone from the application runtime team would like to be notified for `/daml-assistant/` changes?

Reply:
I’ll let you do it in a separate PR since I’m not quite sure if by "under language" you mean everything or only the compiler side.